src/credentials/DeviceAttestationVerifier.cpp

```diff
@@ -52,6 +52,21 @@ class UnimplementedDACVerifier : public DeviceAttestationVerifier
         (void) certDeclBuffer;
         return AttestationVerificationResult::kNotImplemented;
     }
+
+    AttestationVerificationResult ValidateCertificateDeclarationPayload(const ByteSpan & certDeclBuffer,
+                                                                        const ByteSpan & firmwareInfo, uint16_t clusterVendorId,
+                                                                        uint16_t clusterProductId, uint16_t dacVendorId,
```

Comment:
Suggest renames to make it clearer what is being verified:

- clusterProductId -> productId
- clusterVendorId -> vendorId